ethereum_history_api/circuits/lib/src/receipt.nr

```diff
@@ -15,9 +15,9 @@ struct Log<MAX_LOG_DATA_LEN> {
     data: [u8; MAX_LOG_DATA_LEN]
 }
 
-struct TxReceipt<LOG_NUM, MAX_LOG_DATA_LEN> {
-    status: u1,
-    state_root: Bytes32,
+struct TxReceiptPatrial<LOG_NUM, MAX_LOG_DATA_LEN> {
+    status: Option<u1>,
```

Comment:
Why `Option`? On oracle side this is always `'success' | 'reverted'`

Reply:
This is actually an error in viem. receipt did not exist before Byzantium. I'll try to fix it now on JS side